test/unit/validate.test.js

```diff
@@ -39,12 +39,19 @@ describe('Recurly.validate', function () {
     });
 
     it('should parse discover', function () {
-      assert(recurly.validate.cardType('3050990900242198') === 'discover');
-      assert(recurly.validate.cardType('36003010814263') === 'discover');
-      assert(recurly.validate.cardType('6011111111111117') === 'discover');
-      assert(recurly.validate.cardType('6011000990139424') === 'discover');
-      assert(recurly.validate.cardType('6229252986142899') === 'discover');
-      assert(recurly.validate.cardType('8131917256371926354') === 'discover');
+      assert(recurly.validate.cardType('6010999990139424') !== 'discover');
+      assert(recurly.validate.cardType('6011040090139424') !== 'discover');
```

Comment:
What is the actual return value of this call? I'm thinking we should `assert.strictEqual` on the actual return instead of asserting inequality.